XenModel/Messages.resx

```diff
@@ -2214,7 +2214,7 @@ Deleting this bond will automatically transfer the management interface on the b
   <data name="BOND_DELETE_WILL_DISTURB_SECONDARY" xml:space="preserve">
     <value>{0}
 
-Deleting this bond will disrupt traffic through the secondary interface on the bond while the interface is moved to the first slave of the bond.</value>
+Deleting this bond will disrupt traffic through the secondary interface on the bond while the interface is moved to the first other pool member of the bond.</value>

```

Comment:
Apologies, this was skipped accidentally during the first review: "is moved to the first of the bonded members"